lms/djangoapps/certificates/models.py

```diff
@@ -514,6 +514,12 @@ class CertificateInvalidation(TimeStampedModel):
     notes = models.TextField(default=None, null=True)
     active = models.BooleanField(default=True)
 
+    # This is necessary because CMS does not install the certificates app, but it
+    # imports this models code. Simple History will attempt to connect to the installed
```

Comment:
Nit: _model's_

Reply:
Actually, it might be good to update this comment with a description of what broke and why, if we know, since I don't see this model being imported by the CMS